src/App.js

```diff
@@ -20,30 +21,11 @@ class App extends Component {
       user: null
```

Comment:
Usually the default for a property in state should be an empty data type of the data type it will be. This should be an empty object. When deciding whether or not a user exists in your code below, check to see if a user.id exists instead of checking if the whole user exists

Reply:
Thanks for the feedback!